de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/invoicecandidates/impl/CreateInvoiceCandidatesService.java

```diff
@@ -156,6 +155,8 @@ public JsonCreateInvoiceCandidatesResponse createInvoiceCandidates(@NonNull fina
 
 			final InvoiceCandidateId candidateId = externallyReferencedCandidateRepository.save(manualCandidateService.createInvoiceCandidate(candidateToSave));
 
+			externallyReferencedCandidateRepository.storeInvoiceDetailItems(candidateId, candidateToSave.getInvoiceDetailItems());
```

Comment:
Saving the candidate's invoiceDetailItems should be done within or below `externallyReferencedCandidateRepository.save(...)`
I.e. the complete "tree" of the invoice candidate shall be persisted in that method